.travis.yml

```diff
@@ -24,47 +25,12 @@ addons:
             - g++-4.8
 
 before_install:
-    - cd
-    - export INDEXES_PATH=/home/travis/build/indexes
-    - mkdir -p $INDEXES_PATH
-    - TESTED_NODE=$(node -v)
-    - nvm install 0.10
-    - nvm use 0.10
-    - travis_retry npm -g install npm@latest-2
-    - npm install forever coffee-script -g
-    - git clone git://github.com/cozy/cozy-data-system.git
-    - cd cozy-data-system
-    - npm install #data-system
-    - pwd
-    - NAME=data-system TOKEN=token forever start -o forever-ds.log -e forever-ds-err.log build/server.js
-    - sleep 5
-    - pwd
-    - cd
-    - git clone git://github.com/cozy/cozy-proxy.git
-    - cd cozy-proxy
-    - npm install
-    - pwd
-    - NAME=proxy TOKEN=token forever start -o forever-proxy.log -e forever-proxy-err.log build/server.js
-    - sleep 5
-    - 'curl -X POST -H "Content-Type: application/json" -d "{ \"email\": \"cozytest@cozycloud.cc\", \"password\": \"cozytest\", \"timezone\":\"Europe/Paris\"}" http://127.0.0.1:9104/register'
-    - cd
-    - git clone git://github.com/cozy/cozy-files.git
-    - cd cozy-files
+    - bin/install-cozy
```

Comment:
I prefer to keep the install steps in `.travis.yml`.

In practice, I found that the install has failed on many things (travis is full of surprises, like network errors, but the cozy stack has also had its failures) and it's easier to find what has failed when travis runs them step by step, and displayed them step by step in its web interface.